### PR TITLE
update vim ycm installation instruction

### PR DESCRIPTION
**Pull request by PSeitz**
Fixes https://github.com/rust-analyzer/rust-analyzer.github.io/pull/97

### CODE REVIEW

File: docs/user/manual.adoc [hunk 1]
```diff
@@ -255,22 +255,9 @@ let g:LanguageClient_serverCommands = {
 ==== YouCompleteMe
 
 1. Install YouCompleteMe by following the instructions
```
Comment (Veykril) [inline docs/user/manual.adoc:257]: The enumeration here seems unnecessary now

Comment (PSeitz) [reply docs/user/manual.adoc:257]: that's correct

**Comment by lnicola** [comment]
changelog fix Update VIM YCM installation instructions.

**Pull request merged**